transcendence-app/src/shared/db/postgres/utils.ts

```diff
@@ -71,3 +72,18 @@ export function isCriticalDatabaseError(err: unknown): err is DatabaseError {
     (err.severity === 'FATAL' || err.severity === 'PANIC')
   );
 }
+
+export function renameColumnsWithPrefix(
+  tableName: string,
+  columnNames: string[],
+  prefix: string,
+) {
+  return columnNames
+    .map(
+      (name) =>
+        `${tableName}.${name} AS "${prefix}${capitalizeFirstLetter(
+          removeDoubleQuotes(name),
+        )}"`,
+    )
+    .join(', ');
+}
```

Comment:
Thanks to @gbudau who help mw with the query and more

Reply:
Thanks!